### PR TITLE
Remove taosx, connector, explorer and taoskeeper in cloud tdengine

### PR DESCRIPTION
**Pull request by wingwing2005**

### CODE REVIEW

File: packaging/tools/makepkg.sh [hunk 1]
```diff
@@ -69,25 +69,30 @@ if [ "$pagMode" == "lite" ]; then
   bin_files="${build_dir}/bin/${serverName} ${build_dir}/bin/${clientName} ${script_dir}/remove.sh ${script_dir}/startPre.sh ${build_dir}/bin/taosBenchmark "
   taostools_bin_files=""
 else
-
-  wget https://github.com/taosdata/grafanaplugin/releases/latest/download/TDinsight.sh -O ${build_dir}/bin/TDinsight.sh \
+  if [ "$verMode" == "cloud" ]; then
+    taostools_bin_files=" ${build_dir}/bin/taosdump \
```
Comment (sangshuduo) [inline packaging/tools/makepkg.sh]: I guess no need to include taosdump in the cloud instances if we have no interface to download files.

**Review by sangshuduo — APPROVED** [review]
LGTM, with a comment

File: packaging/tools/makepkg.sh (hunk 1 above)
Comment (wingwing2005) [reply packaging/tools/makepkg.sh]: OK, I will correct it.

Comment (wingwing2005) [reply packaging/tools/makepkg.sh]: @sangshuduo have updated the code, please review again.

**Comment by sangshuduo** [comment]
LGTM

**Review by xleili — APPROVED** [review]
LGTM

**Pull request merged**